pkgs/development/python-modules/fabio/default.nix

```diff
@@ -0,0 +1,51 @@
+{
+  lib,
+  buildPythonPackage,
+  ninja,
+  fetchPypi,
+  meson,
+  python3Packages,
```

Comment:
```suggestion
```

You should not use this; the input is already in python3Packages (`python3xxPackages.callPackage`).

Reply:
Ah, didn't know about that. Fixed.